DevIncubator_Autopark/AllCollections/MyStack.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace DevIncubator_Autopark.AllCollections
+{
+    public class MyStack<T> : IEnumerable<T>
+    {
+        public int Count => GetCount();
+
+        private T[] _stack;
+        private int _headItemIndex;
+
+        public MyStack()
+        {
+            int defaultCapacity = 10;
+            _stack = new T[defaultCapacity];
+
+            _headItemIndex = -1;
```

Comment:
Index could not start from negative number. It is illogical.

Reply:
fixed